.cirrus.yml

```diff
@@ -70,7 +70,14 @@ task:
       always:
         format_script: ./script/tool_runner.sh format --fail-on-change
         license_script: dart pub global run flutter_plugin_tools license-check
-        analyze_script: ./script/tool_runner.sh analyze --custom-analysis=script/configs/custom_analysis.yaml
+        pathified_analyze_script:
+          # Run analysis with path-based dependencies to ensure that publishing
+          # the changes won't break analysis of other packages in the respository
+          # that depend on it.
+          - ./script/tool_runner.sh make-deps-path-based --target-dependencies-with-non-breaking-updates
+          - ./script/tool_runner.sh analyze --custom-analysis=script/configs/custom_analysis.yaml
```

Comment:
You're missing https://github.com/flutter/plugins/blob/main/.cirrus.yml#L150-L152 which breaks everything after it.